We Got Coders - My Amazing Application.html~

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```

Comment:
Is this file of relevance? You should be mindful of what you're committing to git. If this is not needed, remember to delete it :+1: